scripts/start.js

```diff
@@ -34,13 +34,21 @@ function start_client(dir) {
 
 // CLI
 (async () => {
+    // Get the start value for the Digital page so that "yarn start" starts on Digital
+    start_value = 0;
+    for (dir of getDirs(true, false)) {
+        if (dir.title === "Digital") {
+            break;
+        }
+        start_value += 1;
+    }
```

Comment:
Nice job! There's actually a really neat way to condense this though in JavaScript using the `findIndex` function, simply:
```js
const start_value = getDirs(true, false).findIndex((dir) => dir.title === "Digital");
```
Also note that in JS you should always declare your variables with `let` or `const` (i.e. `let start_value = 0`) otherwise it has a global scope

Also also, since we're calling `getDirs` twice, we should put at the top, a simple `const dirs = getDirs(true, false);` and reuse that variable